core/consensus/msg.go

```diff
@@ -34,16 +34,20 @@ func newMsg(pbMsg *pbv1.QBFTMsg, justification []*pbv1.QBFTMsg) (msg, error) {
 	var (
 		valueHash         [32]byte
 		preparedValueHash [32]byte
-		err               error
 	)
-	if pbMsg.Value != nil {
-		valueHash, err = hashProto(pbMsg.Value)
+
```

Comment:
The functions in this file seems to be missing a lot of tests. (there are a lot of code coverage warnings).

Reply:
it is covered by integration tests